README.adoc

```diff
@@ -142,8 +142,8 @@ $ odo push
 . Create a URL to access the application and visit it to test it.
 +
 ----
-$ odo url create
-$ curl nodejs-myproject.192.168.42.147.nip.io
+$ odo url create --port <application port>
```

Comment:
port is automatically generated when you do `odo url create` and specifying `--port` should only be used when there are multiple ports to be opened (which `odo` warns the user about...)

Reply:
This is as per input from @surajnarwade Suraj can you please confirm on this?

Reply:
@cdrage ,as of now, `--port` is mandatory, we will try to remove in future but as of now it is mandatory.
because when we do `odo url create`, we just make entry to localconfig, we dont talk to cluster, hence we can not detect the port.